compiler/README.md

```diff
@@ -255,7 +247,7 @@ option that will drop into `lldb` before running.
 
 ### `.lldbinit`
 
-We have a custom `.lldbinit` file in `sorbet_llvm` with some helper commands.
+We have a custom `.lldbinit` file with some helper commands.
```

Comment:
Does that still live in `compiler/` or did it get merged into the toplevel?

Reply:
It’s in the top level (it Changes in this pr)